src/components/Popover/index.tsx

```diff
@@ -117,17 +122,36 @@ export default function Popover({
   const updateCallback = useCallback(() => {
     update && update()
   }, [update])
-  useInterval(updateCallback, show ? 100 : null)
+  useInterval(updateCallback, show ? 200 : null)
+
+  useEffect(() => {
+    if (show) {
+      if (!delayShowTimeout) {
+        setShouldShow(true)
+        return
+      }
+      const tooltipTimer = setTimeout(() => {
+        setShouldShow(true)
+      }, delayShowTimeout)
+
```

Comment:
Move the newline to delineate the two cases:

```suggestion

      const tooltipTimer = setTimeout(() => {
        setShouldShow(true)
      }, delayShowTimeout)
```

Reply:
deleted the delay